cmd/juju/commands/upgrademodel.go

```diff
@@ -375,20 +368,14 @@ func (c *upgradeJujuCommand) getControllerAPI() (ControllerAPI, error) {
 	return apicontroller.NewClient(api), nil
 }
 
+const caasStreamsTimeout = 20 * time.Second
```

Comment:
I assume 20 seconds is enough based on your own testing

Reply:
This was moved from `upgradecontroller.go`.